RallyHereIntegration/Source/RallyHereIntegration/Private/RH_PlatformSessionSyncer.cpp

```diff
@@ -860,8 +860,20 @@ void URH_PlatformSessionSyncer::OnPlatformSessionJoined(EOnJoinSessionCompleteRe
 			{
 				UE_LOG(LogRHSession, Log, TEXT("[%s] - Successfully joined platform session %s"), ANSI_TO_TCHAR(__FUNCTION__), *PlatformSession->GetSessionIdStr());
 
-				auto PlayerUniqueIdPtr = GetOSSUniqueId().GetUniqueNetId();
-				OSSSessionInterface->RegisterPlayer(OSSSessionName, *PlayerUniqueIdPtr.Get(), false);
+				auto PlayerIdWrapper = GetOSSUniqueId();;
+
+				if (PlayerIdWrapper.IsValid())
+				{
+					auto PlayerUniqueId = PlayerIdWrapper.GetUniqueNetId();
+					if (PlayerUniqueId.IsValid())
+					{
+						auto PlayerUniqueIdPtr = PlayerUniqueId.Get();
+						if (PlayerUniqueIdPtr != nullptr && PlayerUniqueIdPtr->IsValid())
```

Comment:
Man Unreal code is very scary with this many chances at nullable redirection

Reply:
This is checking if the pointer is valid, and if the data contained within the struct is valid (it can be an non-null empty struct in some cases, which was causing a crash).